cmd/tools/gendynamicconfig/main.go

```diff
@@ -180,6 +190,16 @@ func (s {{.P.Name}}TypedSetting[T]) Get(c *Collection) func({{.P.GoArgs}}) T {
 		)
 	}
 }
+
+{{if eq .P.Name "Global" -}}
+func GetTypedPropertyFn[T any](value T) TypedPropertyFn[T] {
+{{- else -}}
+func GetTypedPropertyFnFilteredBy{{.P.Name}}[T any](value T) TypedPropertyFnWith{{.P.Name}}Filter[T] {
+{{- end}}
```

Comment:
I left these out because they didn't seem like they'd be commonly used, but no objections if you want them

Reply:
I just added because the other types had.